doc/changes/latest.inc

```diff
@@ -41,6 +41,7 @@ Bugs
 - Fix bug in the ``.compute_psd()`` methods where the number of unaggregated Welch segments was wrongly computed for some inputs, leading to an assertion error when computing the PSD (:gh:`11248` by `Daniel McCloy`_)
 - Fix bug in the :func:`~mne.viz.plot_evoked_topo` and :meth:`~mne.Evoked.plot_topo`, where legend colors where shown incorrectly on newer matplotlib versions (:gh:`11258` by `Erkka Heinila`_)
 - Fix bug where EEGLAB channel positions were read as meters, while they are commonly in millimeters, leading to head outlies of the size of one channel when plotting topomaps. Now ``montage_units`` argument has been added to :func:`~mne.io.read_raw_eeglab` and :func:`~mne.read_epochs_eeglab` to control in what units EEGLAB channel positions are read. The default is millimeters, ``'mm'`` (:gh:`11283` by `Mikołaj Magnuski`_)
+- Fix channel selection edge-cases in `~mne.preprocessing.ICA.find_bads_muscle` (:gh:`11300` by `Mathieu Scheltienne`_)
```

Comment:
tip: you put the changelog entry in the same spot as #11298, guaranteeing yourself a merge conflict.  Next time you're doing 2 PRs at once, feel free to stick one of the entries in the middle of the list; order doesn't matter much for changelog entries (except for keeping `:newcontrib:`s at the top)

Reply:
Yes, I knew I was going to get one, I didn't thought about this simple fix however 😅